libs/platform/src/lib/domain/data-source.ts

```diff
@@ -230,3 +230,30 @@ export class ListDataSource<T> extends ComboBoxDataSource<T> {
     }
 }
 
+export class TableDataSource<T> extends ListDataSource<T> implements DataSource<T> {
+    readonly MaxLimit = 10;
```

Comment:
if intention is to highlight that it is a constant then it should be `MAX_LIMIT`.
`MaxLimit` looks like a reference to a `class` or `type`